src/commands/cmd_stream.cc

```diff
@@ -1715,6 +1684,27 @@ class CommandXReadGroup : public Commander,
     bufferevent_enable(bev, EV_READ);
   }
 
+  static CommandKeyRangeGen KeyRangeGen() {
+    static constexpr auto keyRangeGenFunc = [](const std::vector<std::string> &args) {
```

Comment:
```suggestion
  static CommandKeyRange KeyRangeGen(const std::vector<std::string> &args) {
```

Also we don't need to make a function that returns a function here.

Reply:
Hi @PragmaTwice ,

Got it!

I have changed it to a static variable. 😊

Best Regards,
Edward